e2e/support/test_support.go

```diff
@@ -93,10 +93,10 @@ const kubeConfigEnvVar = "KUBECONFIG"
 
 var TestTimeoutShort = 1 * time.Minute
 var TestTimeoutMedium = 5 * time.Minute
-var TestTimeoutLong = 10 * time.Minute
+var TestTimeoutLong = 15 * time.Minute
 
 // TestTimeoutVeryLong should be used only for testing native builds.
-var TestTimeoutVeryLong = 90 * time.Minute
+var TestTimeoutVeryLong = 60 * time.Minute
```

Comment:
We better provide a specific timeout for the native test, no needs to use global variables. With this change we are slowing down also any other test that would expect those previous values.

Reply:
Hum, actually it is the opposite, I reduced the duration of the timeout by 30 minutes.

Reply:
I was more concerned about `TestTimeoutLong` which is increased a little bit. I wonder if we could instead introduce a different timeout variables for native (ie, `NativeTestTimeoutLong`) or either just use the quantity of minutes. But we refine this in a following iteration, not a blocker for this PR.

Reply:
Yeah, I had to change a bit the timeout because, on MacOS runners, everything is much slower probably because it needs a container runtime which is not the case with Linux. However in the end, If a test works as expected the timeout set should not have any effect on the duration of the test, it is only when something goes wrong the state is not the expected one.